src/buildHTML.js

```diff
@@ -632,6 +635,12 @@ groupTypes.array = function(group, options) {
                 gap = 0;
             }
         }
+        // In AMS multiline environments such as aligned and gathered, rows
+        // correspond to lines that have additional \jot added to the
+        // \baselineskip via \openup.
+        if (group.value.rowsAreLines) {
```

Comment:
The naming of this variable is confusing to me.  Aren't rows already lines?  This just adds more space between the rows/lines.  Maybe change this to `addJotBetweenRows`.